be1-go/hub/standard_hub/mod.go

```diff
@@ -563,9 +563,23 @@ func (h *Hub) createLao(msg message.Message, laoCreate messagedata.LaoCreate,
 		return answer.NewInvalidMessageFieldError("failed to unmarshal public key of the sender: %v", err)
 	}
 
+	organizerBuf, err := base64.URLEncoding.DecodeString(laoCreate.Organizer)
+	if err != nil {
+		return answer.NewInvalidMessageFieldError("failed to decode public key of the organizer: %v", err)
+	}
+
+	organizerPubKey := crypto.Suite.Point()
+	err = organizerPubKey.UnmarshalBinary(organizerBuf)
+	if err != nil {
+		return answer.NewInvalidMessageFieldError("failed to unmarshal public key of the organizer: %v", err)
+	}
+
+	if !organizerPubKey.Equal(senderPubKey) {
+		return answer.NewAccessDeniedError("sender's public key does not match the organizer field: %q != %q", senderPubKey, organizerPubKey)
+	}
+
 	if h.GetPubKeyOwner() != nil && !h.GetPubKeyOwner().Equal(senderPubKey) {
-		return answer.NewAccessDeniedError("sender's public key does not match the organizer's: %q != %q",
-			senderPubKey, h.GetPubKeyOwner())
+		return answer.NewAccessDeniedError("sender's public key does not match the organizer's: %q != %q", senderPubKey, h.GetPubKeyOwner())
```

Comment:
These two error messages are really similar to each other, and may seem like a repeat for someone not knowing well the project. Maybe add a comment or change the error messages (or both)

Reply:
Yes, that's true thanks. I just fixed it in my latest commits.